README.md

```diff
@@ -323,21 +322,32 @@ To create a data source and run it against a specific connection:
 1. Ensure you have at least one folder open in VS Code.
 1. In the **DATASOURCES** View, click **+**. Specify the parameters defined in the following table:
 
-    | Property              | Description                                                             |
-    |-----------------------|-------------------------------------------------------------------------|
-    | Connection            | Select a Connection from the Connections dropdown                       |
-    | Select API            | Choose **getData** from the **Select API** dropdown.                    |
-    | Table                 | Choose the table you wish to query from the **Tables** dropdown.        |
-    | Start Time/End Time   | Select the Start and End Time for the query.                            |
-    | Additional Parameters | You can choose from the additional parameters as required.              |
+    | Property              | Description                                                                                                                     |
+    |-----------------------|---------------------------------------------------------------------------------------------------------------------------------|
+    | Connection            | Select a Connection from list of existing Connections in the dropdown. If none are listed [create a connection](#connections).  |
+    | Select API            | Choose **getData** from the **Select API** dropdown. If none are listed click the **Refresh** button.*                          |
+    | Table                 | Choose the table you wish to query from the **Tables** dropdown.  If none are listed click the **Refresh** button.*             |
+    | Start Time/End Time   | Select the Start and End Time for the query.                                                                                    |
+    | Additional Parameters | You can choose from the additional parameters as required.                                                                      |
+
+    *If you are not already connected when you click the **Refresh** button a notification gives you the option to connect.
+    
+    ![Connection Request](https://github.com/KxSystems/kx-vscode/blob/main/img/connect-request.png?raw=true)
 
+    Once connected you need to click **Refresh** again.
 
 1. Click **Save** to store the settings you have chosen, for reuse later. When you save a data source; query parameters and the connection details are stored. The data source icon is green if it is associated with a connection and grey if there is no association. 
```

Comment:
The datasource icon will only be green if the associated connection is connected.